sdks/python/apache_beam/examples/cookbook/multiple_output_pardo_test.py

```diff
@@ -18,54 +18,43 @@
 """Test for the multiple_output_pardo example."""
 
 import logging
-import re
-import tempfile
 import unittest
 
+import apache_beam as beam
 from apache_beam.examples.cookbook import multiple_output_pardo
+#from apache_beam.transforms.util import assert_that
+#from apache_beam.transforms.util import DataflowAssertException
 
 
-class MultipleOutputParDo(unittest.TestCase):
+class MultipleOutputParDoTest(unittest.TestCase):
 
-  SAMPLE_TEXT = 'A whole new world\nA new fantastic point of view'
-  EXPECTED_SHORT_WORDS = [('A', 2), ('new', 2), ('of', 1)]
-  EXPECTED_WORDS = [
-      ('whole', 1), ('world', 1), ('fantastic', 1), ('point', 1), ('view', 1)]
-
-  def create_temp_file(self, contents):
-    with tempfile.NamedTemporaryFile(delete=False) as f:
-      f.write(contents)
-      return f.name
-
-  def get_wordcount_results(self, temp_path):
-    results = []
-    with open(temp_path) as result_file:
-      for line in result_file:
-        match = re.search(r'([A-Za-z]+): ([0-9]+)', line)
-        if match is not None:
-          results.append((match.group(1), int(match.group(2))))
-    return results
+  SAMPLE_TEXT = 'A whole new world\nA new point'
+  text_len = len(' '.join(SAMPLE_TEXT.split('\n')))
+  SAMPLE_TEXT_Iterable = [SAMPLE_TEXT]
+  EXPECTED_SHORT_WORDS = ['A: 2', 'new: 2']
+  EXP_WORDS = ['point: 1', 'whole: 1', 'world: 1']
 
   def test_multiple_output_pardo(self):
-    temp_path = self.create_temp_file(self.SAMPLE_TEXT)
-    result_prefix = temp_path + '.result'
-
-    multiple_output_pardo.run([
-        '--input=%s*' % temp_path,
-        '--output=%s' % result_prefix])
-
-    expected_char_count = len(''.join(self.SAMPLE_TEXT.split('\n')))
-    with open(result_prefix + '-chars-00000-of-00001') as f:
-      contents = f.read()
-      self.assertEqual(expected_char_count, int(contents))
-
-    short_words = self.get_wordcount_results(
-        result_prefix + '-short-words-00000-of-00001')
-    self.assertEqual(sorted(short_words), sorted(self.EXPECTED_SHORT_WORDS))
-
-    words = self.get_wordcount_results(result_prefix + '-words-00000-of-00001')
-    self.assertEqual(sorted(words), sorted(self.EXPECTED_WORDS))
-
+    p = beam.Pipeline('DirectPipelineRunner')
+    sample_text = p | beam.Create(self.SAMPLE_TEXT_Iterable)
```

Comment:
`beam.Create([self.SAMPLE_TEXT])` could be a simpler way of expressing this, without using another variable.